### DISCUSSION TITLE
how to configure cloudsim v6.0.0 beta sourcecode.zip in eclipse?I am a new user of cloudsim

### DISCUSSION THREAD

**Discussion by MegiTartari3** [Cloudslab/cloudsim#175] [Q&A]
Please help me as I am new user of cloudsim and I need to start experimenting with cloud for my phd studies.
Thank you a lot,Megi

**Comment by itconsultore** [comment]
You could create a new project in Eclipse and use Maven to add the CloudSim libraries.
You can follow the source code of CloudSim examples to personalise the experiments.
